travel.py

```diff
@@ -9,6 +9,44 @@
 sys.setdefaultencoding("utf-8")
 
 
+# 'nick' : ['dep', 'dest', 'time', 'date']
+searched = {}
+
+
+def send_output(nick, dep, dest, date, result=None):
+    """Sends output"""
+    out = 'Nothing found'
+
+    if date == '':
+        date = datetime.datetime.today().strftime('%d.%m.%Y')
+
+    if len(result):
+        out = result[0].__repr__()
+        try:
+            time = result[0].lines[0].departure
+        except AttributeError:
```

Comment:
@Adman this still smells like Java. Can't you just check if the first element of result has a `lines` attribute?